modules/mono/editor/GodotTools/GodotTools/Build/BuildManager.cs

```diff
@@ -350,6 +354,24 @@ public static bool GenerateXCFrameworkBlocking(
             return true;
         }
 
+        private static void AddCustomBuildArguments(BuildInfo buildInfo, bool publish)
+        {
+            string[] arguments = (string[])ProjectSettings.GetSetting("dotnet/build/custom_compiler_arguments");
+            if (publish)
+            {
+                arguments = (string[])ProjectSettings.GetSetting("dotnet/build/custom_publish_arguments");
+            }
```

Comment:
Some users may expect the arguments from `dotnet/build/custom_compiler_arguments` to also apply to the `dotnet publish` command. I wonder if we should append to them, instead of replacing them.